include/Graphs/PAG.h

```diff
@@ -153,6 +153,13 @@ class PAG : public GenericGraph<PAGNode,PAGEdge>
         }
         return pag;
     }
+
+    /// Return pointer to PAG if it has been built, otherwise nullptr.
+    static inline PAG* tryGetPAG()
+    {
+        return pag;
+    }
+
```

Comment:
I don't think we do need this method. You can make the option IncludePAGInICFGDump as false by default. It will solve your problem.

Reply:
It is needed because there is a small window (in the current code) between when the ICFG is made and when the PAG is created. In that interval the ICFG would exist, but the pag would be nullptr. If, in the debugger, you wanted to dump the ICFG, say, just after it has been created, to make sure it is ok, or just to see what it looks like, the ICFG dumper would be executing when the PAG is null, but you would not want the dumper to create the PAG (since a dumper should never have side effects like that). Hence the need for this method (and to avoid complicating the usual getPAG).

Reply:
I don't want to have IncludePAGInICFGDump be false because I want to see the PAG information when it becomes present. I just don't want the ICFG to blow up during the interval between when the ICFG is made and when the PAG shows up (I know in the current code that is a small interval, but one could imagine the code being modified so that the ICFG and PAG were not so closely linked).